quickstarts/microsoft.machinelearningservices/machine-learning-create-vnet/GettingStarted.md

```diff
@@ -0,0 +1,238 @@
+
+Forecasts or predictions from machine learning can make apps and devices smarter. For example, when you shop online, machine learning helps recommend other products you might want based on what you've bought. Or when your credit card is swiped, machine learning compares the transaction to a database of transactions and helps detect fraud. And when your robot vacuum cleaner vacuums a room, machine learning helps it decide whether the job is done.
+
+### What is Azure Machine Learning
+
+Azure Machine Learning provides a cloud-based environment you can use to prep data, train, test, deploy, manage, and track machine learning models. Start training on your local machine and then scale out to the cloud. The service fully supports open-source technologies such as PyTorch, TensorFlow, and sci-kit-learn and can be used for any kind of machine learning, from classical ml to deep learning, supervised and unsupervised learning.
+
+Explore and prepare data, train and test models, and deploy them using rich tools such as:
+
+- A visual interface in which you can drag-n-drop modules to build your experiments and then deploy models
+- Jupyter notebooks in which you use the SDKs to write your code, such as these sample notebooks
+- Visual Studio Code extension
+
+For more information, you can consult [Azure Machine Learning Documentation](https://docs.microsoft.com/en-gb/azure/machine-learning/)
+
+### What is an Azure Machine Learning workspace
+
+The workspace is the top-level resource for Azure Machine Learning, providing a centralized place to work with all the artifacts you create when you use Azure Machine Learning. The workspace keeps a history of all training runs, including logs, metrics, output, and a snapshot of your scripts. You use this information to determine which training run produces the best model.
+
+Once you have a model you like, you register it with the workspace. You then use the registered model and scoring scripts to deploy to Azure Container Instances, Azure Kubernetes Service, or a field-programmable gate array (FPGA) as a REST-based HTTP endpoint. You can also deploy the model to an Azure IoT Edge device as a module.
+
+## The Template
+
+Don't let the size of the template scares you. The structure is very intuitive and once that you get the gist of it, you will see how much easier your life will be deploying resources to Azure.
+
+These are the parameters on the template, they already have values inserted, so you don't need to worry about changing them.
+
+Here the list of all parameters:
+
+Parameter | Suggested value | Description
+:--------------- | :------------- |:---------------------
+**workspaceName** | globally unique name | The name of the machine learning workspace.
+**location** | location | This template takes the location of your Resource Group. Treating appropriately the available locations for each resource.
```

Comment:
it defaults to the RG location but it determines the location of the resources, which can be different than the RG's location